extensions.json

```diff
@@ -32,6 +32,10 @@
       "id": "akamud.vscode-theme-onelight",
       "repository": "https://github.com/akamud/vscode-theme-onelight"
     },
+    {
+      "id": "aki77.rails-db-schema",
+      "repository" : "https://github.com/aki77/vscode-rails-db-schema"
```

Comment:
This extension doesn't have a license, but a valid open source license is required for publishing to OpenVSX. Could you please reach out to the extension owners, and ask them to provide a license?

Reply:
Author has now added license file in the repo - MIT